.github/codemention.yml

```diff
@@ -0,0 +1,232 @@
+  - patterns: ['.github/workflows/**']
```

Comment:
You need a root/top level `rules` key.

Reply:
Good call. Awesome work on the codemention action!